src/main/java/net/openhft/chronicle/bytes/RandomCommon.java

```diff
@@ -216,9 +216,11 @@ Bytes<?> bytesForWrite()
      * @param value    to set
      * @return true, if successful.
      */
+    @Deprecated(/* to be removed in x.25 */)
```

Comment:
Can we say something like `@Deprecated(/* Use this_other_interface_instead, to be removed in x.25 */)`

Reply:
Are these methods implement somewhere else e.g. on one of the output interfaces?

Reply:
yes, they are going to be in output interfaces, however if I add them now, they still show up as deprecated
let me add it since there is no harm
and I'll add more descriptive comment